lib/active_record/connection_adapters/clickhouse/schema_statements.rb

```diff
@@ -8,6 +8,10 @@ def execute(sql, name = nil)
           do_execute(sql, name)
         end
 
+        def execute_with_settings(sql, name = nil, settings: {})
```

Comment:
You do not need to create a duplicate method. You can move `do_execute` to public method.

Reply:
Didn't want to change the original flow :)